src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -6,4 +6,10 @@ Feature: Vodqa test
   Scenario: Validating scroll functionality using 2 points
     Given I login to vodqa application using valid credentials
     When I scroll from one to another element point on vertical swiping screen
-    Then Element text should be visible
\ No newline at end of file
+    Then Element text should be visible
+
+  @android
+  Scenario: Validating scroll functionality using 2 points
+    Given I login to vodqa application using valid credentials
+    When I scroll down by screen size on vertical swiping screen
+    Then Element lower in the list should be visible
```

Comment:
you can directly use "Then Element text should be visible" instead of this as after steps, functionality is almost same

Reply:
removed duplicate step